include/openmc/cell.h

```diff
@@ -52,6 +52,100 @@ extern vector<unique_ptr<Cell>> cells;
 } // namespace model
 
 //==============================================================================
+
+class Region {
+public:
+  //----------------------------------------------------------------------------
+  // Constructors
+  Region() {}
+  explicit Region(std::string region_expressioni, int32_t cell_id);
```

Comment:
To be consistent with the definition
```suggestion
  explicit Region(std::string region_spec, int32_t cell_id);
```